src/js/overlay.js

```diff
@@ -225,16 +219,33 @@ Overlay.prototype.show = function() {
 
 	this.closeOnExternalClick = this.closeOnExternalClick.bind(this);
 	this.delegates.doc.on('click', 'body', this.closeOnExternalClick);
+	this.delegates.doc.on('touchstart', 'body', this.closeOnExternalClick);
```

Comment:
@pornel Ok, will change to touchend then and if adding preventDefault() doesn't trigger the other one, then I'll add it as well even if it's for that small perf gain